iOSTest/View Controllers/LoginViewController.swift

```diff
@@ -70,33 +67,29 @@ class LoginViewController: UIViewController {
     
     func validateUsernameAndPassword() {
 
-        let validUsername = "user@iosstudies.co"
-        let validPassword = "pinksalmon"
-            
+        username = usernameTextfield.text
+        password = passwordTextfield.text
+        
         // Check validity of username and password
         if (username != validUsername) || (password != validPassword) {
             
-            let invalidAlert = UIAlertController(title: "Invalid Username or Password", message: "Please enter a valid username and passowrd.", preferredStyle: .alert)
-            let invalidAction = UIAlertAction(title: "OK", style: .default, handler: nil)
+            let alert = UIAlertController(title: "Invalid Username or Password", message: "Please enter a valid username and passowrd.", preferredStyle: .alert)
+            let action = UIAlertAction(title: "OK", style: .default, handler: nil)
             
-            invalidAlert.addAction(invalidAction)
-            present(invalidAlert, animated: true)
+            alert.addAction(action)
+            present(alert, animated: true)
         }
         
         // Both username and password are valid - go to MenuViewController
         else {
-            let validAlert = UIAlertController(title: "Login Successful", message: "Successfully logged in!", preferredStyle: .alert)
-            let validAction = UIAlertAction(title: "OK", style: .default, handler: {(_) -> Void in self.navigationController!.pushViewController(MenuViewController(), animated: true)})
+            let validAlert = UIAlertController(title: "Login Successful", message: "Function duration: \(runtime)", preferredStyle: .alert)
```

Comment:
Please review instruction #4 at the top of this VC. The ```runtime``` should reflect this request.